pycbc/events/stat.py

```diff
@@ -146,10 +146,7 @@ def coinc_multiifo(self, s, slide, step,
         numpy.ndarray
             Array of coincident ranking statistic values
         """
-        newsnr=0
-        for i in s:
-            newsnr = newsnr + s[i]**2.
-        return newsnr ** 0.5
+        return (sum([s[i]**2 for i in s]))**0.5
```

Comment:
is there a reason not to use ```(sum([x**2 for x in s.values()]))**0.5``` ?  (that would be clearer to me at least)

Reply:
Since the dictionary s is keyed by ifo, for each ifo we take the values from s in the current case. So I would keep the current version, but I can change it if needed.